json/pickler/src/main/scala/sttp/tapir/json/TapirPickle.scala

```diff
@@ -0,0 +1,20 @@
+package sttp.tapir.json
+
+import _root_.upickle.AttributeTagged
+
+trait TapirPickle[T] extends AttributeTagged with Readers with Writers:
```

Comment:
some comments would be good here, as I understand that's our custom uPickle integration, adjusting their defaults so that it serialises things the way we want it to?

Reply:
Yes, I'll add more comments to explain what is what.